cmd/operator/main.go

```diff
@@ -18,15 +18,34 @@ package main
 
 import (
 	"flag"
+	"fmt"
 	"os"
 
 	"github.com/rhobs/observability-operator/pkg/operator"
 	"go.uber.org/zap/zapcore"
 
+	k8sflag "k8s.io/component-base/cli/flag"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 )
 
+// validateImages merges the passed images with the defaults and checks if any
+// unknown image names are passed. If an unknown image is found, this raises an
+// error.
+func validateImages(images *k8sflag.MapStringString) (map[string]string, error) {
+	res := operator.DefaultImages
```

Comment:
(nit) I would move DefaultImages to the main package because there's no use of it in pkg/operator

Reply:
+1 to that, as it would be more clear, but totally a nit